mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -356,7 +353,7 @@ function resultsToTables(results, target, uri) {
     tableid = "#" + tableid;
     $(target).html(html);
     $(target).attr('data', uri);
-    //retreive albumcovers
+    //retrieve albumcovers
     for (i = 0; i < coversList.length; i++) {
         getCover(coversList[i][0], target + '-cover-' + coversList[i][1], 'small');
```

Comment:
Oh now I get it. Event so, at the moment only the call from ``resultsToTables`` will have more than one URI available for calling `` mopidy.getImages`` on, so we can probably leave that for another PR as you suggested.

I'm still a little confused as to why the images are being retrieved here to start with, as some of the tables being rendered in the client don't even have the ability to display images as far as I can tell.